doc/content/getting-started/platform-guide/_index.md

```diff
@@ -5,27 +5,87 @@ weight: -1
 distributions: null
 ---
 
-The {{% gn %}} platform is designed to provide an easy way to get started with device development.
+The {{% gn %}} platform is designed to provide an easy way to get started with device development. It comes pre-programmed with a demo app. 
+<!--more-->
+ {{% gnse %}} is currently in the production phase. If you don't have one get yours at [thethingsshop.com](https://www.thethingsshop.com/products/generic-node-sensor-edition)! Please follow these steps to claim your new device and start using it right away.
 
 <!--more-->
 
-To get started with the {{% gn %}} platform, you typically need two things:
 
-1. {{% gn %}} supported hardware
-2. {{% gn %}} supported software
+## Step 1: Create a new account on The Things Stack (TTS) Cloud
+
+We have a dedicated The Things Stack Cloud tenant for the {{% gn %}}. Please head to this url and create a new account if you don't have one. 
+You may also sign in with your Things ID account: [Generic Node TTS Cloud](https://gnse.eu1.cloud.thethings.industries/). You're also free to use our community stack ([The Thigns Network](https://eu1.cloud.thethings.network/)) or any other TTS Cloud tenant for this purpose.
+
+
+## Step 2: Select your region and create a new Application
+
+Create new Application with a unique app-id. Adding the app name and the description are optional.  
+{{< figure src="gnse_claim_0.png" alt="Create new Application" >}}
+
+
+Once you have a test applicatin similar to this we can move to the next step:
+{{< figure src="gnse_claim_1.png" alt="Test application" >}}
+
+
+## Step 3: Claim your device ownership
+
+{{% gnse %}} comes pre-flashed from the factory which the vanilla app that uses the on-board secure element to join the LoRaWAN network using over the air activation (OTAA) method.
+
+In order to register your device and claim its ownership all you have to do is to use information that's printed on the back of each device during the end device registration process. 
+
+We click on **Register end device** button to create a new one. Here we have two options:
```

Comment:
Avoid "we" and just make a statement - "click on the button to ...". And which two options do you mean here?